tools/scons/site_scons/site_tools/emscripten/emscripten.py

```diff
@@ -6,6 +6,14 @@
 
 from __future__ import print_function
 import os
+import sys
+
+
+def get_building_env_variables(emscripten_path):
+  sys.path.append(emscripten_path)
+  import tools.building
```

Comment:
I don't love that idea of importing parts of the emscripten python code/library into scons like.  I'd rather think of scons as a separate tool, and I'd rather it not be aware of the emscripten internals like this.  

Can you revert or split out this part of the change?